manage_people/templates/manage_people/add_user_confirmation.html

```diff
@@ -26,7 +26,11 @@ <h3>
                     <span class="label">HKL</span>
                 {% endif %}
                 <div class="float-right">
-                    {% if existing_enrollment %}
+                    {% if enrollee.error_message %}
```

Comment:
I believe Django messages is enabled in this project and referenced in a base template. Might be better or easier instead of as an attribute of an enrollee